devel/libfilezilla/Portfile

```diff
@@ -19,8 +19,9 @@ long_description    Small and modern C++ library, offering some basic \
 homepage            https://lib.filezilla-project.org/
 master_sites        http://download.filezilla-project.org/libfilezilla/
 
-checksums           rmd160  d5ff932d4d8088429940d1041d943bf9055a6c44 \
-                    sha256  ecbaa674c0ad0b63df842b8cde17935a497dd58c3749baa281c67cf5878e64f7
+checksums           rmd160  8d9011d64d6577493e2924f6661ef1e42a99f617 \
+                    sha256  96e59946fcf6f9444abfb4aea9f9275b9fa66f868bed00571bc32898bb178472 \
+                    sha512  a2242fd67a054d47a938011c07a074f7e6d4a8fd7352b88411db1302d24a908a7b39ea76cd38cfe63765eb522476da4ccc22164884f9563e263a9e0895e7217f
```

Comment:
`sha512` checksum is not supported in MacPorts yet.

https://github.com/macports/macports-base/blob/v2.4.2/src/port1.0/portchecksum.tcl#L55